### ISSUE TITLE
modals/sidepanel - better accessibility -  undismissable alert dialogs

### ISSUE THREAD

**Issue by lucienbertin** [LuccaSA/lucca-front#1137]
for accessibility, there are two kind of dialog : `dialog` and `alertDialog`. the first one is optional and should be dismissable ; while the second one is mandatory and the end user must make a choice

LF should provide tools to use these concepts, for that

- add the correct aria role or attr to the modal/sp panel
- have the correct aria stuff for the buttons that dismiss this modal/sp
- provide a suite of tools to open alertDialogs, which is not currently available un LF

**Comment by sanlucca** [comment]
In review ? @vvalentin-lucca @lucienbertin did any of you work on that subject ?

**Comment by lucienbertin** [comment]
not yet no
